templates/Dockerfile.rhel

```diff
@@ -20,7 +20,7 @@ LABEL name="nuxeo/nuxeo" \
 # install java
 RUN yum-config-manager --disable rhel-7-server-htb-rpms && \
     yum install -y \
-       java-1.8.0-openjdk java-1.8.0-openjdk-devel wget unzip
+       java-11-openjdk wget unzip
```

Comment:
Don't we need a full JDK and not a simple JRE? or did the RHEL packaging changed?

Reply:
There's no more JRE.